vllm/entrypoints/llm.py

```diff
@@ -140,13 +140,15 @@ def set_tokenizer(
 
     def generate(
         self,
-        prompts: Optional[Union[str, List[str]]] = None,
+        prompts: Optional[Union[str, List[str], List[Dict[str, str]],
+                                List[List[Dict[str, str]]]]] = None,
         sampling_params: Optional[Union[SamplingParams,
                                         List[SamplingParams]]] = None,
         prompt_token_ids: Optional[List[List[int]]] = None,
         use_tqdm: bool = True,
         lora_request: Optional[LoRARequest] = None,
         multi_modal_data: Optional[MultiModalData] = None,
+        chat_template: Optional[str] = None,
```

Comment:
I don't think we want to change the `generate` entrypoint

Reply:
I changed it back, at first I thought it would be helpful for generate to accept chat interface.